packages/testnet-faucets/src/ts/dispense_asset_tasks.ts

```diff
@@ -32,17 +33,22 @@ export const dispenseAssetTasks = {
             logUtils.log(`Sent ${DISPENSE_AMOUNT_ETHER} ETH to ${recipientAddress} tx: ${txHash}`);
         };
     },
-    dispenseTokenTask(recipientAddress: string, tokenSymbol: string, zeroEx: ZeroEx): AsyncTask {
+    dispenseTokenTask(
+        recipientAddress: string,
+        tokenSymbol: string,
+        networkId: number,
+        erc20TokenWrapper: ERC20TokenWrapper,
+    ): AsyncTask {
         return async () => {
             logUtils.log(`Processing ${tokenSymbol} ${recipientAddress}`);
             const amountToDispense = new BigNumber(DISPENSE_AMOUNT_TOKEN);
-            const token = await zeroEx.tokenRegistry.getTokenBySymbolIfExistsAsync(tokenSymbol);
+            const token = TOKENS_BY_NETWORK[networkId][tokenSymbol];
```

Comment:
Is the `networkId`'s received validated to make sure they exist within this mapping? Otherwise we might get some `x is undefined` errors here. Safer would be to do:

```
const tokenIfExists = _.get(TOKENS_BY_NETWORK, [networkId, tokenSymbol]);
if (_.isUndefined(tokenIfExists)) {
...
```